actions.yaml

```diff
@@ -38,8 +38,43 @@ create-admin-account:
     phone_number:
       description: The admin's phone number
       type: string
-  # TODO: Remove required password once password resetting is implemented
```

Comment:
This can't be done because according to [docs](https://www.ory.sh/docs/kratos/manage-identities/import-user-accounts-identities)
> when you import identities, you must provide the credentials field.

Reply:
Let's add this in the PR description!